templates/install/troubleshoot.md

```diff
@@ -0,0 +1,29 @@
+## Troubleshooting Common Problems
+
+In this page we document a set of common problems that users encounter when using their lean toolchain installation. It does not deal with issues that users may encounter with a proof or a language feature. This webpage is a constant Work in Progress. Users who are unable to find a solution to their problems on this page are encouraged to describe their problem on [Zulip](https://leanprover.zulipchat.com/)
+
+### Lost your Infoview
+
+You can get it back as follows
+* On a PC or laptop which typically runs windows/linux press `Ctrl+Shift+Enter`
+* On an apple machine press `Cmd+Shift+Enter`
+
+### Get Access to Docs inside VSCode
+
+You can get the Lean documentation inside VS Code using `Ctrl-Shift-p` (`Cmd-Shift-p` on MacOS) and then, inside the text field that appears, type `lean doc`` and hit Enter. You will be led to the documentation viewer on a side panel.
+
+### Command Not Found Error on MacOS or Linux
+If you encounter any command not found errors when opening a new terminal, try the following quick-fixes in order:
+* If you are on macOS and have not tinkered with your shell you are most likely using a shell called `zsh`. Run the command `source ~/.zshrc` and try again.
+* If you are on linux, currently the default shell in popular distributions like Ubuntu or Linux Mint is `bash`. In this case, run `source ~/.bashrc`.   
+* If this still doesn't work, logging out and logging in again should fix it. 
+
+### InitializeSecurityContext error on Windows
```

Comment:
Having this information in two different places feels like a recipe for confusion to me.

Reply:
Perhaps we could link to this paragraph from other pages. In such pages we have a section titled "Errors you might encounter" and link them to this section.